packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -4133,12 +4135,22 @@ class EditableTextState extends State<EditableText> with AutomaticKeepAliveClien
     _textInputConnection!.setComposingRect(composingRect);
   }
 
+  // Sends the current caret rect to the embedder's text input plugin.
+  //
+  // The position of the caret rect is updated on each change such that if the
```

Comment:
nit: based on the current implementation, it's sent periodically (instead of on each change)?

Reply:
Updated the text.